### DISCUSSION TITLE
Is it not possible to edit memory from the Inspector?

### DISCUSSION THREAD

**Discussion by awh-tokyo** [dirkwhoffmann/virtualc64#764] [Q&A]
When the emulator is paused, I can double-click on a line of memory from the inspector and it turns into editable text, but as soon as I hit enter, the contents go back to what they were before I started the edit.  Is there any way to edit memory "live"?

**Comment by dirkwhoffmann** [comment]
At the moment, there is no way to edit memory. The data view in the memory panel was falsely labelled as "editable" in Xcode. It should not be possible to modify the displayed data. In vAmiga it's possible to alter memory in the RetroShell debugger, but this functionality hasn't been ported to VirtualC64 yet.

**Reply by freaky-m0** [reply]
Is it "easy" to port this feature? ;) :)
